src/prefect/client/subscriptions.py

```diff
@@ -42,7 +42,12 @@ async def __anext__(self) -> S:
 
                 await self._websocket.send(orjson.dumps({"type": "ack"}).decode())
 
-                return self.model.parse_raw(message)
+                data = orjson.loads(message)
+
+                if (v := data.get("type")) and v == "ping":
+                    continue
```

Comment:
Ahhh yep I did change the protocol slightly.  This is here to handle the case of an older Prefect server before I made that change, right?

Reply:
If so, I think the right approach would be to send a `pong` message back, but maybe we just shouldn't support that anymore?  This was all under an experimental flag so we aren't making any backwards compatibility guarantees yet